src/time_machine.py

```diff
@@ -33,6 +34,19 @@ def time(self):
             _time_machine.original_time() - self.real_start_timestamp
         )
 
+    def tick(self, delta: dt.timedelta = DEFAULT_TICK_DELTA):
```

Comment:
no type hint, this is incorrect even

Also no need for a var, it's only used in one place

```suggestion
    def tick(self, delta=dt.timedelta(seconds=1)):
```